src/cli/config.rs

```diff
@@ -0,0 +1,62 @@
+use crate::{api_client::Client, types::StateId};
+use clap::{Args, Parser, Subcommand};
+use std::fmt;
+
+#[derive(Debug, Parser)]
+#[clap(version, about = "Beacon API client")]
+pub struct CliConfig {
+    #[clap(short, long)]
+    pub endpoint: String,
+    #[clap(subcommand)]
+    pub command: Namespace,
+}
+
+#[derive(Debug, Clone, Subcommand)]
+#[clap(author, version, about)]
+pub enum Namespace {
+    #[clap(subcommand)]
+    Beacon(BeaconMethod),
+}
+
+impl fmt::Display for Namespace {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let printable = match *self {
+            Namespace::Beacon(_) => "beacon",
+        };
+        write!(f, "{printable}")
+    }
+}
+
+#[derive(Debug, Clone, Subcommand)]
+pub enum BeaconMethod {
+    //Beacon ns
+    Genesis(GenesisArg),
```

Comment:
this method doesn't have any args

https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getGenesis

```suggestion
    Genesis,
```